packages/help-center/src/components/help-center-search-results.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable no-restricted-imports */
```

Comment:
Is this ESLint ignore still necessary?

Reply:
This file imports a bunch of things from `client`... which is quite bad AFAIK. An npm package shouldn't import things from the consumer of that package. It looks like there's a ton of this pattern in this package (many added in https://github.com/Automattic/wp-calypso/pull/65329 cc @arcangelini @alshakero). Is that safe?

Reply:
Agreed and we've been having some conversations to improve that, since we caught these while working on https://github.com/Automattic/wp-calypso/issues/74399. Is this something @Automattic/vertex folks can follow up on @alshakero?